.devops/nix/package.nix

```diff
@@ -65,6 +65,9 @@ let
     strings.optionalString (suffices != [ ])
       ", accelerated with ${strings.concatStringsSep ", " suffices}";
 
+  crossCompileWindows = (effectiveStdenv.buildPlatform.isLinux || effectiveStdenv.buildPlatform.isDarwin) && effectiveStdenv.targetPlatform.isMinGW;
+  executableSuffix = strings.optionalString (crossCompileWindows) ".exe";
```

Comment:
Isn't it enough to test just `isMinGW` now? Also it's actually enough to only test the `hostPlatform` I believe

Reply:
Ah yes, as we only use this variable for checking which extension to use it is enough to just check our build target. I don't quite understand why you would use `hostPlatform` instead of `buildTarget` though?

Reply:
H'm? `hostPlatform` is the one on which we'll be running the resulting `llama.exe`. Coincidentally, `targetPlatform` will be the same in all meaningful instantiations of nixpkgs, but I think `hostPlatform.isMinGW` is the one we want?

Reply:
Comitted